src/functionalTest/java/uk/gov/hmcts/reform/fact/functional/helpers/TestDataHelper.java

```diff
@@ -30,6 +30,19 @@ public static String getRegionId(final HttpClient http) {
         return response.jsonPath().getString("[0].id");
     }
 
+    /**
+     * Fetches a specific opening hour type ID by index from the opening hour types endpoint.
+     * Valid indices are 0-8 for the 9 available opening hour types.
+     *
+     * @param http the HTTP client
+     * @param index the index of the opening hour type (0-8)
+     * @return the opening hour type ID at the specified index as a UUID
+     */
+    public static UUID getOpeningHourTypeId(final HttpClient http, final int index) {
+        final Response response = http.doGet("/types/v1/opening-hours-types");
```

Comment:
Interesting. Why out of interest do we not reuse the existing GET method where you pass in a URL (lets say) and then extract what you need from the response that gets returned from the general method?